app/src/main/java/de/hhn/aib/labsw/blackmirror/controller/API/Websockets/MirrorApiWebsockets.kt

```diff
@@ -86,9 +93,7 @@ class MirrorApiWebsockets : WebSocketListener(),MirrorApi {
      * @param payload payload of the message, not null
      */
     override fun publish(topic: String, payload: Any) {
-        val sendPackage = SendPackage()
-        sendPackage.topic = topic
-        sendPackage.payload = mapper.valueToTree(payload)
+        val sendPackage = SendPackage(topic, getJSONMapper().valueToTree(payload))
```

Comment:
replace `getJSONMapper().xxx` with `mapper.xxx` when using the Kotlin property syntax (getter properties instead of getter methods)